.github/ISSUE_TEMPLATE/release.md

```diff
@@ -13,33 +13,34 @@ labels: ["release"]
 - [ ] Ensure all PRs in the k6 repository, part of the current [milestone](https://github.com/grafana/k6/milestones), have been merged.
 - [ ] Open a PR with the release notes for the new version, and ask teams who might have contributed to the release (@k6-browser, @k6-chaos, @devrel teams, etc.) to contribute their notes and review the existing ones.
 - [ ] Share the release notes PR with the k6 open-source teams. Request contributions from all affected teams (browser, chaos, devrel, docs, etc.) and any other stakeholders involved in the new release.
 - [ ] Open a separate PR for bumping [the k6 Go project's version](https://github.com/grafana/k6/blob/9fa50b2d1f259cdccff5cc7bc18a236d31c345ac/lib/consts/consts.go#L11).
-- [ ] Create a dedicated branch for the upcoming version in the grafana/k6-DefinitelyTyped fork repository.
-- [ ] Open a PR in the DefinitelyTyped/DefinitelyTyped repository, using the branch created in the grafana/k6-DefinitelyTyped fork, to update the k6 type definitions for the new release.
-
+- [ ] Open a PR in the `DefinitelyTyped/DefinitelyTyped` repository, using the release branch created in the grafana/k6-DefinitelyTyped fork, to update the k6 type definitions for the new release.
 
 #### ~ 1 day before the release date.
 
-- [ ] Open a PR in the k6-docs repository, archiving the current k6's JavaScript API version as per the following [instructions](https://github.com/grafana/k6-docs/wiki/Add-version-for-Javascript-API-documentation).
-- [ ] Ensure the [existing k6-docs PRs](https://github.com/grafana/k6-docs/pulls), related to the new functionalities and changes, are reviewed, up to date with the latest state of the `master` branch, and based upon the branch containing the k6 archived JavaScript API documentation (as created in the previous step).
+- [ ] Open a PR from `main` in the `k6-docs` repository, archiving the current k6's JavaScript API version as per the following [instructions](https://github.com/grafana/k6-docs/wiki/Add-version-for-Javascript-API-documentation).
+- [ ] Open a PR in `k6-docs` repository for the release branch containing all the merged PRs for the upcoming version. If any, resolve the conflicts with `main` branch.
 
 ### Release Day
 
 #### Documentation
 
-- [ ] Merge the k6-docs repository's Javascript API archiving PR and rebase the rest of the branches meant for the release on top of the new state of the `master` branch.
-- [ ] Merge all the k6-docs repository's branches containing changes related to the release.
+- [ ] Merge the `k6-docs` repository's Javascript API archiving PR and rebase the rest of the branches meant for the release on top of the new state of the `main` branch.
+- [ ] Merge the release PR for `k6-docs` containing all the changes related to the release. The order with the the previous PR is relevant otherwise the new changes will be included in the previous versions.
```

Comment:
```suggestion
- [ ] Merge the release PR for `k6-docs` containing all the changes related to the release. The order in which this PR is merged relative to the previous one is crucial; otherwise, the new changes might inadvertently be incorporated into previous versions.
```